script/installation/packages.sh

```diff
@@ -180,149 +190,25 @@ if [ "$DISTRO" = "UBUNTU" ]; then
 ## DEBIAN
 ## ------------------------------------------------
 elif [ "$DISTRO" = "DEBIAN OS" ]; then
-    sudo apt-get -q --ignore-missing -y install \
-        git \
-        g++ \
-        clang \
-        cmake \
-        libgflags-dev \
-        libprotobuf-dev \
-        protobuf-compiler \
-        bison \
-        flex \
-        libevent-dev \
-        libboost-dev \
-        libboost-thread-dev \
-        libboost-filesystem-dev \
-        libjemalloc-dev \
-        libssl-dev \
-        valgrind \
-        lcov \
-        libpqxx-dev \
-        llvm-dev \
-        libedit-dev \
-        postgresql-client \
-        libtbb-dev \
-        libeigen3-dev
+    echo "Error: Debian is not currently supported"
+    exit 1
 
 ## ------------------------------------------------
 ## FEDORA
 ## ------------------------------------------------
 elif [[ "$DISTRO" == *"FEDORA"* ]]; then
```

Comment:
Instead of keeping Fedora and RedHat in the if clauses, why not just let it fall down to the unsupported clause and let it spit the error message.

We don't want people to think that we will support it.